code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -195,9 +198,38 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	cell_type = /obj/item/stock_parts/cell/gun/mini
 	ammo_type = list(/obj/item/ammo_casing/energy/lasergun/hitscan)
+	manufacturer = MANUFACTURER_SHARPLITE_NEW
 
 /obj/item/gun/energy/laser/hitscanpistol/examine_more(mob/user)
 	if(in_range(src, user) || isobserver(user))
-		. = list("<span class='notice'>You examine [src] closer. Under the grip is a small inscription: \"NT CN SVALINN 2502\".</span>")
+		. = list("<span class='notice'>You examine [src] closer. Under the grip is a small inscription: \"NT CN SVALINN 462\".</span>")
 	else
 		. = list("<span class='warning'>You try to examine [src] closer, but you're too far away.</span>")
+
+/obj/item/gun/energy/laser/e10
+	name = "E-10 laser pistol"
+	desc = "A very old laser weapon. Despite the extreme age of some of these weapons, they are sometimes prefered to the mass produced Nanotrasen laser weapons."
```

Comment:
Spelling edit.

"A very old laser weapon. Despite the extreme age of some of these weapons, they are sometimes preferred to newer, mass-produced Nanotrasen laser weapons."